packages/sysutils/busybox/scripts/init

```diff
@@ -61,6 +61,10 @@
 
   LIVE="no"
 
+  # Get a serial number if present (eg. RPi) otherwise use MAC address from eth0
+  MACHINE_UID="$(cat /proc/cpuinfo | awk '/^Serial/{s=$3; gsub ("^0*","",s); print s}')"
```

Comment:
`sed -n 's|^Serial.*: ||p' /proc/cpuinfo` also works
or
`sed -n 's|^Serial.* \(0.*\)|\1|p' /proc/cpuinfo` which is less fragile.